MediaBrowser.Common/Json/Converters/JsonGuidConverter.cs

```diff
@@ -13,21 +13,13 @@ public class JsonGuidConverter : JsonConverter<Guid>
         public override Guid Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
         {
             var guidStr = reader.GetString();
-
             return guidStr == null ? Guid.Empty : new Guid(guidStr);
         }
 
         /// <inheritdoc />
         public override void Write(Utf8JsonWriter writer, Guid value, JsonSerializerOptions options)
         {
-            if (value == Guid.Empty)
-            {
-                writer.WriteNullValue();
-            }
-            else
-            {
-                writer.WriteStringValue(value);
-            }
+            writer.WriteStringValue(value);
```

Comment:
Will this stop all the 0000000-000-000 guids?

Reply:
This writes the Guid.Empty values, if we don't want a Guid to be sent it should be marked as nullable, and the NullableStructConverter will handle it

Reply:
See what you've done now.